app/models/preorder_information.rb

```diff
@@ -27,13 +27,13 @@ def initialize(*args)
 
   # If this method is added, we may need to update School::SchoolDetailsSummaryListComponent
   def infer_status
-    if school_will_order_devices? && school_contact.nil?
+    if school_will_order_devices? && !any_school_users? && school_contact.nil?
       'needs_contact'
-    elsif school_will_order_devices? && school_contacted_at.nil?
+    elsif school_will_order_devices? && !any_school_users? && school_contact.present?
       'school_will_be_contacted'
-    elsif school_will_order_devices? && school_contacted_at.present? && !chromebook_information_complete?
+    elsif school_will_order_devices? && any_school_users? && !chromebook_information_complete?
       'school_contacted'
```

Comment:
If I'm reading this right, then we're no longer using the `school_contact` field to infer whether the school has been contacted or not. It's now primarily whether the school has any users or not. 

So is "school will be contacted" / "school_contacted" still the right set of names for these two states? (changing the names could of course be a separate PR)

Reply:
I think we will be able to retire the "school will be contacted" status – it's currently used to park:

- schools we've not contacted yet until the service can scale to handle the volume
- schools we can't contact yet because we can't add users to multiple schools yet

After we solve both, we should revisit the statuses and pare stuff down...